Bugfix/zenko 4505 fix pensieve dataconsumer and multiplebackend tagging

williamlardier opened the pull request:
Bump component versions to include the following fixes:

- [PSVAPI-41](https://scality.atlassian.net/browse/PSVAPI-41)
- [CLDSRV-343](https://scality.atlassian.net/browse/CLDSRV-343)
- [VAULT-497](https://scality.atlassian.net/browse/VAULT-497)

[PSVAPI-41]: https://scality.atlassian.net/browse/PSVAPI-41?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CLDSRV-343]: https://scality.atlassian.net/browse/CLDSRV-343?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[VAULT-497]: https://scality.atlassian.net/browse/VAULT-497?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Note for reviewers: I just rebased to change the order of commits, here is a green build with all the changes, if you review this PR whie it is still building: https://github.com/scality/Zenko/commit/1efaf4fc30d0816cb6f939ab7e4f903bff596313

Note 2: the 2nd commit disables swagger validation in the bravado python client we use in this CI, as it behave differently than our Swagger validation in other CIs.

Seragonia committed:
ZENKO-4505: bump cloudserver to 8.7.8, pensieve to 1.4.3 and vault to 8.7.4

Seragonia committed:
ZENKO-4505: bump Zenko to 2.6.11

bert-e commented:
# Hello williamlardier,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://bert-e.scality.net/scality/zenko/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4505 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.6.11`


Please check the `Fix Version/s` of ZENKO-4505, or the target
branch of this pull request.

Seragonia committed:
ZENKO-4505: Disable Swagger spec validation

The validation of the swagger spec by the bravado client is
returning an error for a required (but unused) paramater, while
the swagger validation in prior CI steps was not blocking.

ghost commented:
/approve

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4505 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.6.11`


Please check the `Fix Version/s` of ZENKO-4505, or the target
branch of this pull request.




*The following options are set:* **approve**

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue ZENKO-4505 contains:


* *None*



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `2.6.11`


Please check the `Fix Version/s` of ZENKO-4505, or the target
branch of this pull request.




*The following options are set:* **approve**

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/2.6`



The following branches will **NOT** be impacted:

* `development/2.3`
* `development/2.4`
* `development/2.5`



There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/2.6`



The following branches have **NOT** changed:

* `development/2.3`
* `development/2.4`
* `development/2.5`



Please check the status of the associated issue ZENKO-4505.

Goodbye williamlardier.
